eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/docbook.iso.sch

```diff
@@ -246,6 +246,7 @@
     <s:rule context="db:link">
       <s:assert test="not(.//db:link)">link must not occur in the descendants of link</s:assert>
       <s:assert test="not(.//db:ezlink)">ezlink must not occur in the descendants of link</s:assert>
+      <s:assert test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))">using scripts in links is not allowed</s:assert>
```

Comment:
@pspanja Wouldn't it be better to do this during validation? So like here but maybe try to do in`rng` to see if a better error message can be given to user?

Or where you suggesting to aim to silently ignore it?